src/message/MQMessage.cpp

```diff
@@ -109,9 +112,9 @@ void MQMessage::setPropertyInternal(const string& name, const string& value) {
 
 const string & MQMessage::getProperty(const string& name) const {
```

Comment:
@ifplusor hi, if we can change getProperty return string but not string& ? because i don't think there is a need to return reference string for getProperty

Reply:
@jonnxu I think 'const string&' is ok. Here, we need return a read-only object or a new one, and return a reference don't allocate a new object.

Reply:
I think both of them are fine.
It will not affect efficiency too much.